src/utils/diff/colors.ts

```diff
@@ -0,0 +1,24 @@
+import { Colors } from './types';
```

Comment:
It's better to integrate this into the existing color management system and use HTML in the diffs. Check this:
https://github.com/DevExpress/testcafe/blob/master/src/reporter/plugin-host.js#L44
https://github.com/DevExpress/testcafe/blob/32c140881e60d955239d29219ff9ad717a236c59/src/errors/test-run/utils.js#L82